docs/synapse/userguides/ug015_storm_ref_pivot.rst

```diff
@@ -321,6 +323,8 @@ N/A
 
 **Usage notes:**
 
+* ``totags()`` does consume nodes by design.
```

Comment:
Neutral on whether we need this - assumption to date has been that an operator consumes nodes unless otherwise specified.

Reply:
removed